ably/rest_channel_test.go

```diff
@@ -187,4 +196,299 @@ func TestRestChannel(t *testing.T) {
 			}
 		}
 	})
+
+}
+
+func TestIdempotentPublishing(t *testing.T) {
+	t.Parallel()
+	app, err := ablytest.NewSandboxWIthEnv(nil, "idempotent-dev")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer app.Close()
+	options := app.Options()
+	client, err := ably.NewRestClient(options)
```

Comment:
I don't see any of these tests specify `NoIdempotentRestPublishing`. I realise that it defaults to `true`, but I would prefer it to be explicit. It also means there are no tests with `NoIdempotentRestPublishing=false`.

Reply:
well unless I missed your point. That code has `NoIdempotentRestPublishing=false`.